node-notes-cli/readNote.js

```diff
@@ -0,0 +1,13 @@
+const data = require('./data.json');
+
+function readNote() {
+  for (let i = 1; i < data.nextId; i++) {
+    for (const note in data.notes) {
+      if (parseInt(note) === i) {
+        console.log(`${note}: ${data.notes[note]}`);
+      }
+    }
```

Comment:
the `nextId` isn't a counter or anything, it's only there for creation of each note. You can use a `for...in` loop to print each note without any conditional statement.